packages/mask/src/plugins/Avatar/types.ts

```diff
@@ -10,6 +10,8 @@ export interface AvatarMetaDB {
     chainId?: ChainId
     schema?: SchemaType
     pluginId?: NetworkPluginID
+    contractName?: string
```

Comment:
For flow we need to encode the contract name into the address. Since then, we don't really have to store this somewhere.

Reply:
It means if the token comes from the hub API. The token address already encoded with a contract name in it.